commands/command_edit.h

```diff
@@ -424,6 +424,21 @@ class EditCylinder : public EditCylinderBase {
 	void redo() override;
 };
 
+class EditSensors : public EditDivesBase
```

Comment:
I think that `EditDivesBase` is the wrong base class here. I would model it after the `ReplanDive` command. Then these multiple- vs. one-dive issues should go away.